tests/stress/enc/enc.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/internal/print.h>
+#include "stress_t.h"
+
+void do_ecall(int arg)
+{
+    // almost do nothing, just assign value
+    int rcv = 0;
```

Comment:
I suspected that the compiler might flag this as an unused variable and the bors try run confirms it; you'll need to figure out a more concrete action for the ecall, for example, incrementing a static global.

From the [jenkins ](https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/pipelines%2FAzure-Linux/detail/Azure-Linux/680/pipeline) run:

```
[ 85%] Building CXX object tests/stress/enc/CMakeFiles/stress_enc.dir/enc.cpp.o

/home/oeadmin/workspace/pipelines/Azure-Linux/tests/stress/enc/enc.cpp: In function ‘void do_ecall(int)’:

/home/oeadmin/workspace/pipelines/Azure-Linux/tests/stress/enc/enc.cpp:10:9: error: variable ‘rcv’ set but not used [-Werror=unused-but-set-variable]

     int rcv = 0;

         ^~~
```

Reply:
Have added a static global to use the argu brought in.